### DISCUSSION TITLE
connection to server at "postgres" (172.20.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres" Stopping docker

### DISCUSSION THREAD

**Discussion by shiwam77** [serverpod/serverpod#938] [Q&A]
D:\studioprojectD\muddra\muddra_server> .\setup-tables.cmd                
Starting docker
[+] Running 3/3
 ✔ Network muddra_server_default       Created                                                                                                                                     0.7s 
 ✔ Container muddra_server-redis-1     Started                                                                                                                                     2.0s 
Waiting for Postgres...
Postgres is ready
failed to get console mode for stdin: The handle is invalid.
psql: error: connection to server at "postgres" (172.20.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres"
Stopping docker

**Comment by Isakdl** [comment]
What did you do to reach this?

Perhaps you have an old volume from a previous container with the same name?

Take a look at: https://stackoverflow.com/questions/76172995/serverpod-unable-to-authenticate-with-postgres

**Comment by raghav042** [comment]
I resolved this error and posted an answer on stackoverflow: https://stackoverflow.com/a/77252351/14360762
